scripts/creating_mutation_summary_table.R

```diff
@@ -31,6 +31,11 @@ create_summary <- function ( files, output_file ){
 }
 
 args <- commandArgs (trailingOnly=TRUE)
+
+cat("\n\"")
+cat(paste(args, collapse = "\",\n\""))
+cat("\"\n\n")
+
```

Comment:
When I look at these similar changes I thought to myself:

- would be nice to have this as a shared procedure in pigx-common (or wherever)
- would be nice to make printing conditional (PIGX_VERBOSE env var?)